src/views/CashlinkReceive.vue

```diff
@@ -0,0 +1,682 @@
+    }
+
+    private goToOnboarding() {
+        this.callHub('onboard');
+    }
+
+    private callHub(method: string) {
+        const request: BasicRequest = {
+            appName: 'Cashlink',
+        };
+
+        const redirectBehavior = new HubApi.RedirectRequestBehavior(undefined, { url: window.location.href });
+        const hubApi = new HubApi(undefined, redirectBehavior);
+
+        switch (method) {
+            case 'signup': hubApi.signup(request); break;
+            case 'login': hubApi.login(request); break;
+            case 'onboard': hubApi.onboard(request); break;
+        }
+    }
+
+    private redirectToSafe() {
+        window.location.href = Config.network === NETWORK_MAIN
+            ? 'https://safe.nimiq.com'
+            : 'https://safe.nimiq-testnet.com';
+    }
+
+    private get isCashlinkStateKnown(): boolean {
+        if (!this.consensus) return false;
+        return this.cashlink!.state !== CashlinkState.UNKNOWN;
+    }
+
+    private get canCashlinkBeClaimed(): boolean {
+        if (!this.consensus) return false;
+        return this.cashlink!.state === CashlinkState.UNCLAIMED;
+    }
+
+    private get buttonText(): string {
+        if (!this.isCashlinkStateKnown) return 'Checking status';
+        else if (this.canCashlinkBeClaimed) return 'Claim cashlink';
+        else if (this.cashlink!.state === CashlinkState.UNCHARGED) return 'Cashlink not funded';
+        else if (this.cashlink!.state === CashlinkState.CHARGING) return 'Cashlink funding';
+        else {
+            if (!this.isClaiming) {
+                this.statusState = StatusScreen.State.WARNING;
+                this.statusTitle = 'Cashlink is empty';
+                this.statusMessage = 'This cashlink has already been claimed.';
+            }
+            return 'Cashlink empty :(';
+        }
+    }
+
+    private get isButtonLoading(): boolean {
+        return !this.isCashlinkStateKnown || this.cashlink!.state === CashlinkState.CHARGING;
+    }
+
+    private get isCashlinkAlreadyClaimed(): boolean {
+        return this.cashlink!.state >= CashlinkState.CLAIMING;
+    }
+}
+</script>
+
+<style scoped>
+    .container {
+        flex-direction: row !important;
+        padding: 0 5rem; /* Side padding for smaller screens */
+    }
+
+    .card-content {
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
+    }
+
+    .card-content.no-account {
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    .welcome-text {
+        max-width: 514px;
+        margin-left: 8.75rem; /* nq-card already has 1.25rem margin */
+        transition: opacity .8s, transform .8s;
+    }
+
+    .welcome-text .nq-h1 {
+        font-size: 8rem;
+        margin-top: 0;
+        margin-bottom: 4rem;
+    }
+
+    .welcome-text .nq-text {
+        font-size: 4rem;
+        color: rgba(31, 35, 72, 0.5);
+    }
+
+    .welcome-text .main-text {
+        color: var(--nimiq-blue);
```

Comment:
[Optional]
Could be `--nimiq-blue` on `.welcome-text` and `opacity: .5` on `.welcome-text .nq-text`.

Reply:
`.main-text` is _inside_ `.nq-text`. So setting opacity on the outside one cannot be undone on an inside child.

Reply:
I changed it to give the reduced opacity to the secondary text now. However, I did not use `opactiy`, because that also makes the highlight color transparent.